pkgs/servers/authelia/default.nix

```diff
@@ -2,13 +2,21 @@
   lib,
   stdenv,
   nodejs,
-  pnpm,
+  fetchPnpmDeps,
+  pnpmConfigHook,
   fetchFromGitHub,
   buildGoModule,
   installShellFiles,
   callPackage,
   nixosTests,
-  authelia-web ? callPackage ./web.nix { inherit nodejs pnpm fetchFromGitHub; },
+  authelia-web ? callPackage ./web.nix {
+    inherit
+      nodejs
+      fetchPnpmDeps
+      pnpmConfigHook
+      fetchFromGitHub
+      ;
+  },
```

Comment:
This removed `pnpm`, but seems like the package still builds.
I'm not sure why are only some of the arguments listed here while others are inferred like `stdenv`, I imagine it's because of overriding, which is probably only needed for `nodejs` and `pnpm`.